paddle/operators/math/math_function.cc

```diff
@@ -130,6 +132,99 @@ void matmul<platform::CPUPlace, double>(
       matrix_b.data<double>(), beta, matrix_out->data<double>());
 }
 
+template struct SetConstant<platform::CPUPlace, float>;
```

Comment:
Let us move the added code into `paddle/operators/math/selected_rows.{h,cc,cu}`.

Reply:
Done